src/SIL.LCModel/Infrastructure/Impl/SharedXMLBackendProvider.cs

```diff
@@ -262,7 +262,7 @@ private void CreateSharedMemory(bool createdNew)
 
 		private MemoryMappedFile CreateOrOpen(string name, long capacity, bool createdNew)
 		{
-			if (MiscUtils.IsMono)
+			if (MiscUtils.IsMono || !MiscUtils.IsWindows)
```

Comment:
Would `SIL.PlatformUtilities.Platform.IsUnix` (alone) work here? If not, `!Platform.IsWindows` alone should work. Both are in `SIL.Core`. Either of these <img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>